index.js

```diff
@@ -21,4 +21,21 @@ $(document).ready(function() {
         autoplaySpeed: 2000,
         speed: 1000,
     });
-});
\ No newline at end of file
+});
+
+let acc = document.getElementsByClassName("accordion");
+let i;
```

Comment:
Why not declare inline in the loop on line 29?

Reply:
you are right